docs/robots/ridgeback/user_manual_ridgeback.mdx

```diff
@@ -1,12 +1,330 @@
+| Pulsing green       | Pulsing green          | Charger is connected, Ridgeback's battery is charging. Note that the battery can also be charged with the platform off. This indication will only be seen if the charger is connected while Ridgeback is powered up.                                                                                                                                                                                                            |
+| Solid green         | Solid green            | Charger is connected and Ridgeback is charged fully.                                                                                                                                                                                                                                                                                                                                                                            |
+| Solid white         | Solid red              | Normal operation; ready to drive. The intensity of the lights will increase slightly when actually in motion. This is the status which may be overridden by publishing your own light patterns to the `/cmd_lights` ROS topic.                                                                                                                                                                                                  |
+
+#### User Bay
+
+The User Bay provides access to the User Power Panel, as well as USB and Ethernet ports, and additional
+space for stowing user equipment. The electrical panel can be used to power your payloads, with further details available
+in [Electrical Integration](/docs/robots/ridgeback/integration_ridgeback#ridgeback-payload-electrical-integration).
+The USB3 ports are connected directly to the onboard computer, and the ethernet ports connect to the onboard network switch. To
+connect a device to the onboard network, it is suggested to give it a static IP in the 192.168.131.x/24 subnet and
+this cannot be changed since the MCU is statically set to 192.168.131.2 and expects the computer to be at
+192.168.131.1. Avoid IPs in use by the following pre-existing devices:
+
+| IP Address     | Device Assignment                              |
+| :------------- | :--------------------------------------------- |
+| 192.168.131.1  | Onboard PC (both ports, br0 network interface) |
+| 192.168.131.2  | Ethernet-connected MCU                         |
+| 192.168.131.20 | Front-facing LIDAR                             |
+| 192.168.131.21 | Rear-facing LIDAR (if present)                 |
+
```

Comment:
You could add a link to our common IPs page:
/docs/tools/network_ip_addresses

Reply:
Updated